t/lib/t/MusicBrainz/Server/Controller/Admin/DeleteEditor.pm

```diff
@@ -34,6 +43,11 @@ test 'Delete account as a regular user' => sub {
     html_ok($mech->content);
     $mech->content_contains('You need to be logged in to view this page');
 
+    $second_session->get('/');
+    is($second_session->status, 500, 'restoring deleted user fails');
+
+    $second_session->get_ok('/');
+    $second_session->content_contains('Log In', 'no longer logged in in second session');
```

Comment:
If indeed someone does it, the session will have been invalidated, so just reloading the page works.